### PR TITLE
Add Bootstrap and custom logo to clean up front end

### PR DESCRIPTION
**Pull request by margaretchan**
old screenshot: JWKyM746gBB

### CODE REVIEW

**Review by josefj1519 — APPROVED** [review]

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -1,32 +1,42 @@
...
-                <tr>
-                    <td><a href="youtube.com">youtube.com</a></td>
-                    <td>XXXXX</td>
-                </tr>
-            </table>
-        </div>
+                    <tr>
+                        <!-- This is an example row -->
+                        <td><a href="youtube.com">youtube.com</a></td>
+                        <td>XXXXX</td>
```
Comment (josefj1519) [inline src/main/webapp/index.html]: Get rid of the placeholders.

Comment (margaretchan) [reply src/main/webapp/index.html]: Removed.

**Comment by margaretchan** [comment]
Updated screenshot: ConWcmTnzEm

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -1,32 +1,36 @@
...
-<html>
+<html lang="en">
     <head>
         <meta charset="UTF-8">
         <title>PromoTube</title>
         <link rel="stylesheet" href="style.css">
-            <script src="script.js"></script>
+        <script src="script.js"></script>
+        <link href='https://fonts.googleapis.com/css?family=Oswald' rel='stylesheet'>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
```
Comment (derrillhd) [inline src/main/webapp/index.html:9]: what's this integrity thing?... is it necessary?

**Review by derrillhd — APPROVED** [review]

File: src/main/webapp/index.html (hunk 2 above)
Comment (margaretchan) [reply src/main/webapp/index.html:9]: Not really sure... just copied from the website [instructions](https://getbootstrap.com/docs/4.3/getting-started/download/).

**Pull request merged**